scipy/stats/tests/test_multivariate.py

```diff
@@ -216,7 +216,7 @@ def test_broadcasting(self):
                 # Repeat for cdf
                 actual = multivariate_normal.cdf(X[i, j], mean, cov)
                 desired = multivariate_normal.cdf(X, mean, cov)[i, j]
```

Comment:
small optimization
AFAICS `desired` arrays only need to be computed once outside of the loop.

As a broadcasting test it might only need some "nice" x, in case there are values that are difficult to compute or have tiny cdf values.